app/models/document.rb

```diff
@@ -59,4 +59,8 @@ def user_facing_state
   def title_or_fallback
     title.presence || I18n.t!("documents.untitled_document")
   end
+
+  def topics
+    DocumentTopics.new(self)
```

Comment:
Since this isn't memo'd doesn't this mean we do a HTTP request each time you call this method?

Reply:
I don't think so - the memoisation is on the specific methods of the DocumentTopics class, which I now notice I didn't add yet (was postponing it to the final commit). Thanks for spotting :-).

Reply:
You'd still get a new instance of DocumentTopics though so that wouldn't have access to the memoisation of the previous one?

Reply:
Oops again - fixed!